  File "/home/ros_dev/workspace/track/install/track/lib/python3.10/site-packages/track/depth.py", line 20
    msg.data[:2] = 
                   ^
SyntaxError: invalid syntax

